### DISCUSSION TITLE
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'F:\\'

### DISCUSSION THREAD

**Discussion by Kapowpenguin1** [AlUlkesh/stable-diffusion-webui-images-browser#183] [Q&A] [answered]
I recently moved my stable diffusion directory to a new drive, the webui seems to start up fine, but image browser is giving me errors, and the tab doesn't appear. any tips on what to do? idk if this is relevant, but i also reinstalled Git for Windows

**Comment by AlUlkesh** [comment]
Can you post the errors in more detail?

**Reply by Kapowpenguin1** [reply]
\>Image Browser: ImageReward is not installed, cannot be used.
\>Loading weights [0fc198c490] from D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\models\Stable-diffusion\AbyssOrangeMix2_hard.safetensors
\>Creating model from config: D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\configs\v1-inference.yaml
\>LatentDiffusion: Running in eps-prediction mode
\>DiffusionWrapper has 859.52 M params.
\>Loading VAE weights specified in settings: D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\models\Stable-diffusion\nai.vae.pt
\>Applying xformers cross attention optimization.
\>Textual inversion embeddings loaded(1): awaitingtongue
\>Model loaded in 2.2s (load weights from disk: 0.4s, create model: 0.4s, apply weights to model: 0.5s, apply half(): 0.6s, load VAE: 0.4s).
\>*Deforum ControlNet support: enabled*
\>CUDA SETUP: Loading binary D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cudaall.dll...
\>Error executing callback ui_tabs_callback for D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py
\>Traceback (most recent call last):
\>>  File "D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\modules\script_callbacks.py", line 125, in ui_tabs_callback
    res += c.callback() or []
\>>  File "D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py", line 1634, in on_ui_tabs
    create_tab(tab, current_gr_tab)
\>>  File "D:\Executables\Stable Diffusion WebUI\stable-diffusion-webui-gpu\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py", line 1122, in create_tab
    os.makedirs(dir_name)
\>>  File "C:\Users\[username]\AppData\Local\Programs\Python\Python310\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
\>>  File "C:\Users\[username]\AppData\Local\Programs\Python\Python310\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
\>>  File "C:\Users\[username]\AppData\Local\Programs\Python\Python310\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  [Previous line repeated 3 more times]
\>>  File "C:\Users\[username]\AppData\Local\Programs\Python\Python310\lib\os.py", line 225, in makedirs
    mkdir(name, mode)
\>FileNotFoundError: [WinError 3] The system cannot find the path specified: `'F:\\'`

[">" added for readability]

**Reply by AlUlkesh** [reply]
What are your webui settings for the various paths? This:

![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/99896447/62200903-1774-42ef-a388-9ac5f3a3e0ab)

**Reply by Kapowpenguin1** [reply]
![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/12951389/df1f39a5-94c6-4451-bd83-3158e250d607)
nothing looks off...

**Comment by AlUlkesh** [comment]
Can you get me a full debug log? As described here: #130

**Reply by Kapowpenguin1** [reply]
unfortunately, this process requires me to be able to access the Image Browser tab, which is simply not appearing for me rn

**Reply by Kapowpenguin1** [reply, marked as answer]
wait, nvm, i see a "image_browser.txt" file and it seems to have all the relevant info. it seems like i forgot to update "List of active tabs" with the new path, since i added a custom directory to it

yep, updated the path, and now image browser works fine
